2-numbers.js

```diff
@@ -0,0 +1,62 @@
+//Task 1 - Determine whether a variable is odd or even
+var oddNum = 13;
+var evenNum = 10;
+
+var checkNum = function (num) {
+    if (num % 2 === 0) {
+        console.log(num + " is even");
+    }
+    else {
+        console.log(num + " is odd");
+    };
+};
+
+checkNum(evenNum);
+checkNum(oddNum);
+
+//Task 2 - Convert Celsius into Farenheit
+var temp = 20;
+
+var convertCelToFah = function (temp) {
```

Comment:
Don't fix this time, unless I make later comments - we use full explicit naming:
convertCelciusToFarenhiet